datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -57,21 +149,56 @@ export default function SchemaHeader({
     hasKeySchema,
     showKeySchema,
     setShowKeySchema,
+    lastUpdatedTimeString,
+    selectedVersion,
+    semanticVersionList,
+    schemaView,
+    setSchemaView,
 }: Props) {
+    const history = useHistory();
+    const location = useLocation();
     const onVersionChange = (version1, version2) => {
         if (version1 === null || version2 === null) {
             return;
         }
         fetchVersions?.(version1 - maxVersion, version2 - maxVersion);
     };
 
+    const semanticVersionDisplayString = (semanticVersion: SemanticVersionStruct) => {
+        const semanticVersionTimestampString =
+            (semanticVersion?.semanticVersionTimestamp &&
+                toRelativeTimeString(semanticVersion?.semanticVersionTimestamp)) ||
+            'unknown';
+        return `${semanticVersion.semanticVersion} - ${semanticVersionTimestampString}`;
+    };
+
+    const renderOptions = () => {
+        return semanticVersionList.map(
+            (semanticVersionStruct) =>
+                semanticVersionStruct?.semanticVersion &&
+                semanticVersionStruct?.semanticVersionTimestamp && (
+                    <SchemaBlameSelectorOption
+                        value={semanticVersionStruct?.semanticVersion}
+                        data-testid={`sem-ver-select-button-${semanticVersionStruct?.semanticVersion}`}
+                    >
+                        {semanticVersionDisplayString(semanticVersionStruct)}
+                    </SchemaBlameSelectorOption>
+                ),
+        );
+    };
+
+    const onSchemaBlameToggle = (e) => {
+        setSchemaView(e.target.value);
+    };
+
+    const docLink = 'https://datahubproject.io/docs/dev-guides/timeline/';
```

Comment:
Nit: this should probably in a constants file outside of the component

Reply:
(not required to address right now but in future cases keep this in mind.)

Reply:
Good point, will do in the future